examples/magnesium_pbe.jl

```diff
@@ -48,17 +48,12 @@ for key in sort([keys(energies)...]; by=S -> string(S))
 end
 @printf "\n    %-20s%-15.12f\n\n" "total" sum(values(energies))
 
-# Band structure calculation along high-symmetry path
-kpoints, klabels, kpath = high_symmetry_kpath(basis, kline_density, composition...)
-println("Computing bands along kpath:\n     $(join(kpath[1], " -> "))")
-band_data = compute_bands(ham, kpoints, n_bands)
+# Plot band structure
+plot_bands(ham, n_bands, kline_density, composition, scfres.εF)
 
-# Plot bandstructure using pymatgen
-plotter = pyimport("pymatgen.electronic_structure.plotter")
-bs = pymatgen_bandstructure(basis, band_data, klabels, fermi_level=scfres.εF)
-bsplot = plotter.BSPlotter(bs)
-plt = bsplot.get_plot()
-plt.autoscale()
-plt.savefig("magnesium_pbe.pdf")
-plt.legend()
-plt.show()
+# Plot DOS
+εs = range(minimum(minimum(scfres.orben))-1, maximum(maximum(scfres.orben))+1, length=1000)
+Ds = DOS.(εs, Ref(basis), Ref(scfres.orben), T=Tsmear*4, smearing=DFTK.smearing_methfessel_paxton_1)
```

Comment:
I'd prefer one space around the operators ...

On that note: I think we should start to settle on some syntax agreements like that at some point and write it down somewhere.

Reply:
Sure. Very far at the bottom of my priority list though. This is the kind of things that can be done after the fact with automatic formatting tools. There are a couple of active ones: https://github.com/julia-vscode/DocumentFormat.jl https://github.com/domluna/JuliaFormatter.jl

Reply:
Awesome ... I did not know there was something in the pipeline :smile: